client/build.gradle

```diff
@@ -44,3 +44,8 @@ task bundle(type: NpmTask) {
   args = ['run', 'build']
   dependsOn generate
 }
+
+// set the version in package.json to the same version as the gradle project
+task updateVersionFromGradle(type: NpmTask) {
+  args = ['version', rootProject.version.toString()]
```

Comment:
I think we should add `-no-git-tag-version` here so that gradle stays in charge of the tag.